client/elements/addons/sc-static-page.html

```diff
@@ -36,7 +36,7 @@
               };
               setTimeout(() => {
                       this.currentId = this._scrollToSection(window.location.hash, this.currentId);
-              }, 1000);
+              }, 1500);
```

Comment:
Try using `requestAnimationFrame` instead of `setTimeout`, it might just work.

Reply:
No, tried it but it doesn't.